### DISCUSSION TITLE
Python version

### DISCUSSION THREAD

**Discussion by dheerajchand** [CoderLine/alphaTab#1154] [Ideas]
Is there any plan for an official Python release? It would be nice to be able to generate charts and interact with them through frameworks like Django or to be able to use the various statistical tools in Python for aleatoric music and write charts out.

**Comment by Danielku15** [comment]
> Is there any plan for an official Python release? 

At this point I would say: currently rather no. Technically it would be possible to also create a Python target, but as for the other platforms it requires long term maintenance and documentation. Development wise it would be similar to the other cross compilations like C# and Kotlin. I think there is no special syntax or mechanism we use, which wouldn't be available on python.

If the interest of a python version would grow, I could certainly add it. I have some ideas in mind to make it less effort to maintain all the different platforms but didn't yet find to look into this.
